addons/binding/org.openhab.binding.autelis/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,568 @@
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="system-opmode">
+		<item-type>Number</item-type>
+		<label>Operation Mode</label>
+		<description>The current state of the Intellitouch® controller</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Auto</option>
+				<option value="1">Service/Timeout</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-freeze">
+		<item-type>Number</item-type>
+		<label>Freeze State</label>
+		<description>The state of freeze protect</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">Off</option>
+				<option value="1">On</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-sensor1">
+		<item-type>Number</item-type>
+		<label>Sensor 1</label>
+		<description>The state of sensor 1 (water sensor)</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">OK</option>
+				<option value="1">ERROR</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-sensor2">
+		<item-type>Number</item-type>
+		<label>Sensor 2</label>
+		<description>The state of sensor 2 (solar sensor)</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">OK</option>
+				<option value="1">ERROR</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-sensor3">
+		<item-type>Number</item-type>
+		<label>Sensor 3</label>
+		<description>The state of sensor 3 (air sensor)</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">OK</option>
+				<option value="1">ERROR</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-sensor4">
+		<item-type>Number</item-type>
+		<label>Sensor 4</label>
+		<description>The state of sensor 4 (water sensor 2)</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">OK</option>
+				<option value="1">ERROR</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-sensor5">
+		<item-type>Number</item-type>
+		<label>Sensor 5</label>
+		<description>The state of sensor 5 (solar sensor 2)</description>
+		<state readOnly="true">
+			<options>
+				<option value="0">OK</option>
+				<option value="1">ERROR</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="system-version">
+		<item-type>String</item-type>
+		<label>Version</label>
+		<description>The firmware version of the Pool Control device </description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="system-time">
+		<item-type>Number</item-type>
+		<label>Time</label>
+		<description>The time as kept by the Pool Control device</description>
+		<state readOnly="true">
+		</state>
+	</channel-type>
+	<channel-type id="equipment-circuit1">
+		<item-type>Switch</item-type>
+		<label>Circuit 1</label>
+		<description>The current state of circuit 1 (Spa or Lo-Temp)</description>
```

Comment:
Why does circuit 1 and 6 have a different desc than the others (additional "Spa or Lo-Temp")?